src/react/components/_dashboard/TasksCard/CompletedTasks/ByCategory.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { connect } from 'react-redux';
+
+import {
+  groupByAidYear,
+  checklistCategoryTitles,
+  groupByCategory,
+} from '../tasks.module';
+
+import Category from '../Category';
+import Tasks from '../Tasks';
+import Agreement from './Agreement';
+
+import SectionHeader from '../SectionHeader';
+import CompletedTask from './CompletedTask';
+import CategoryTitle from './CategoryTitle';
+
+// Completed tasks are organized by "Category"
+// The financial aid category is broken up by Aid Year, further into groups of
+// "incomplete" and "being processed".
+const ByCategory = ({ categories }) => {
+  return (
+    <>
```

Comment:
@developish - What does `<>` do?

Reply:
It's a shortcut for <Fragment> so a component can return sibling elements. I tend to prefer the more verbose version. Updated.